sdk/trace/config.go

```diff
@@ -50,6 +51,12 @@ func (sl *SpanLimits) ensureDefault() {
 	}
 }
 
+func (sl *SpanLimits) parsePotentialEnvConfigs() {
+	sl.AttributeCountLimit = env.IntEnvOr(env.SpanAttributesCountKey, sl.AttributeCountLimit)
+	sl.LinkCountLimit = env.IntEnvOr(env.SpanLinkCountKey, sl.LinkCountLimit)
+	sl.EventCountLimit = env.IntEnvOr(env.SpanEventCountKey, sl.EventCountLimit)
```

Comment:
Can you add the new env vars to the GoDoc?

Reply:
Added